html-forms/index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="http://127.0.0.1:5500/html-forms/index.html">
+      <label>
+        <div>Email</div>
+        <input type="email" name="Email" id="email" />
+      </label>
+    </form>
+    <form action="http://127.0.0.1:5500/html-forms/index.html">
+      <div>
+        <label for="why is coding cool">Why is coding cool</label>
+      </div>
+      <select name="Why is coding cool" id="why is coding cool">
+        <option value="Select and option">Select an Option</option>
```

Comment:
when the option is used for a default message, you do not need to include a value for it